xbmc/video/videosync/VideoSyncCocoa.cpp

```diff
@@ -142,12 +125,19 @@ void CVideoSyncCocoa::OnResetDevice()
 
 float CVideoSyncCocoa::GetFps()
 {
+  int fpsInt;
 #if defined(TARGET_DARWIN_IOS)
-  UpdateFPS(g_Windowing.GetDisplayLinkFPS() + 0.5);
+  fpsInt = MathUtils::round_int(g_Windowing.GetDisplayLinkFPS() + 0.5);
+
+  if (fpsInt != MathUtils::round_int(m_fps))
+  {
+    m_fps = fpsInt;
+  }
 #else
-  UpdateFPS(Cocoa_GetCVDisplayLinkRefreshPeriod());
+  m_fps = g_graphicsContext.GetFPS();
+  fpsInt = MathUtils::round_int(m_fps);
```

Comment:
Here it's just for display.  It doesn't store it that way.  And even then, it's only because it was rounded for display before.  I can change that if you like.